src/components/form/form-field.ts

```diff
@@ -0,0 +1,45 @@
+import { FormField } from 'src/utils/form/form-field/form-field';
+import { DirectiveOptions, VNode, VNodeDirective } from 'vue';
+
+let touchFormField: any;
+
+export const FormFieldDirective: DirectiveOptions = {
+    inserted(
+        el: HTMLElement,
+        binding: VNodeDirective,
+        vnode: VNode
+    ): void {
+        const formField: FormField<any> = binding.value;
+        touchFormField = () => formField.touch();
+
+        el.addEventListener('blur', touchFormField, true);
+    },
+    update(
+        el: HTMLElement,
+        binding: VNodeDirective,
+        vnode: VNode
+    ): void {
+        const formField: FormField<any> = binding.value;
+
+        if (formField.shouldFocus) {
+            if (el instanceof HTMLInputElement) {
```

Comment:
Ici un quick dirty hack qui fait mieux le travail =>

const container: HTMLDivElement = new HTMLDivElement();
container.appendChild(this.$el);
container.querySelector...

Reply:
brillant, j'essaye  ça